app/models/common/datetime.py

```diff
@@ -29,26 +23,6 @@ class Date(BaseModel):
         le=31
     )
 
-    @validator('day')
-    def check_date(cls, v, values):
```

Comment:
Why sis you remove this?  This check was useful and the `leap_year` should still be a property of the date.

Reply:
@chriswmackey the only place `Date` was being used in the schema was for ScheduleFixedInterval and the `is_leap_year` property is being repeated there https://github.com/ladybug-tools-in2/energy-model-schema/blob/d6c57ecc86b5bf92b2129a945cac32c3ba30fad8/app/models/energy/schedulefixedinterval.py#L46